app/_posts/blog/2016-05-29-2016-retrospective-planning-for-2017.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "DEVit 2016 Retrospective, planning for the future"
+description: In this article we go through the DEVit 2016 big moments, fails and all the excitement we experienced, and then we get a sneak peak into what 2017 has in hold for us.
+metaTitle: DEVit 2016 Retrospective, planning for the future
+metaDescription: In this article we go through the DEVit 2016 big moments, fails and all the excitement we experienced, and then we get a sneak peak into what 2017 has in hold for us.
+category : blog
+tags : [retrospective, devit2016, devit2017]
+author: Thanasis Polychronakis
+---
+
+
+## DEVit 2016 is over!
+
+Oh what a blast DEVit 2016 was! No matter how exhausted we were we could not stop the rush of endorphins that flooded our bodies for days after. You see, we, and by 'we' I'm referring to everyone that was at DEVit, managed to make a difference, we participated in making history for our community, for our country. We have successfully positioned Greece in the global landscape of international conferences, helped make DEVit a sustainable yearly conference and had such an amazing time!
+
+We know we messed a few things up, I'll get to them one by one in a bit, but what prevailed in the end was our amazing attitude, passion and determination to have a good time. Your amazing vibes gave us all the motivation we needed to continue on with DEVit, scale it and present you with an even more amazing experience.
+
+## Your feedback, the bad staff
+
+As I said before we did mess a few things up... At registration we definitely dropped the ball, at our retrospective meeting we dived deep into all the various reasons why this happened, they were a lot and we had to deal with multiple failures even with some of our contingency plans. Be rest assured that we didn't take this lightly, we have already devised a new game plan and will work hard towards making the registration process as fast and seamless as possible.
```

Comment:
At registration we definitely dropped the ball and we dived deep into all the various reasons why this happened.
[ at our retrospective meeting] <-- no need for this imho
We had to deal with multiple failures even **on** some of our contingency plans. Be rest assured that we didn't take this lightly, we have already devised a new game plan and will have a seamless registration process from now on.